app/views/associateucr/associate_ucr_summary.scala.html

```diff
@@ -27,11 +29,11 @@
         linkContent: linkContent,
         formHelper: uk.gov.hmrc.govukfrontend.views.html.components.FormWithCSRF)
 
-@(associateUcr: AssociateUcr, mucr: String)(implicit request: Request[_], messages: Messages)
+@(associateUcr: AssociateUcr, mucr: String)(implicit request: JourneyRequest[_], messages: Messages)
 
 @govukLayout(
     title = Title("associate.ucr.summary.title"),
-    backButton = Some(BackButton(messages("site.back"), consolidations.routes.AssociateUcrController.displayPage)),
+    backButton = Some(BackButton(messages("site.back"), AssociateUcrSummaryController.backlink)),
```

Comment:
Uhm :thinking: May I say that I do not like too much the view calling a controller's method? Why not placing the method directly in the view? If you need to also use elsewhere the method a views/helpers is a better place imo

Reply:
I did it this way because it's somewhat analogous to the way we have `Navigator` abstracted out in exports UI.
But at the same time I didn't want to create a whole new class for one backlink, so I opted for an object in the controller.

I generally think our views are not deciding on navigation?

Reply:
I mean technically it's not the controller class but the companion object.

Reply:
In same way they do that. If you think, once the form is submitted they redirect to the controller.

Reply:
I do not understand... why you would need to create a new class for the backlink!?

Reply:
As in I didn't want to go to the length of creating a `Navigator` for movements frontend.